src/Views/Movie/Create.cshtml

```diff
@@ -0,0 +1,46 @@
+@using Microsoft.AspNetCore.Mvc.Localization
```

Comment:
This file's okay, but it could do a little better on spacing (specially after commas)